.tekton/windows-machine-config-operator-bundle-master-pull-request.yaml

```
@@ -29,6 +29,10 @@ spec:
     value: Containerfile.bundle
   - name: git-url
     value: '{{source_url}}'
+  - name: mergeTargetBranch
+    value: true 
+  - name: targetBranch
+    value: "master"
```

The example uses target_branch from GitHub events.  This should allow the code to be the same for all branches and not be specific to each branch.
```suggestion
    value: "{{ target_branch }}"
```
As stated in the pipeline yaml:
```
spec:
  params:
    # The variable with brackets are special to Pipelines as Code
    # They will automatically be expanded with the events from Github.
```